webui/ec2/EC2.py

```diff
@@ -130,3 +145,62 @@ def create_user(username):
     u = User(user.user_name, user.user_id, user.create_date,
              [AccessKey(key.id, key.status, key.create_date, key.secret)])
     return u
+
+
+def list_instances(region='eu-central-1'):
+    ec2 = boto3.resource('ec2', aws_access_key_id=provider_conf.EC2['key'],
+                         aws_secret_access_key=provider_conf.EC2['secret'],
+                         region_name=region)
+    return ec2.instances.all()
+
+
+def create_csp_info(i):
```

Comment:
name is misleading , you not creating something here  you just converting boto3 to json, feel free to come up with different one : 
```suggestion
def boto3_to_json(boto3_obj):
```